### PR TITLE
Characteristics - Stats / Skills / Talents Backend, /datum/character backend, background-locked alt title backend

### PR DESCRIPTION
**Pull request by silicons**
# This entire PR is config-disabled and is just going to add backend, character setup, etc. You are not affected yet.

Implementation will come later.
The costs however, will be finalized here.

See code/modules/mob/characteristics/readme.md for more information.

tl;dr

- bay skills
- numeric + raw stats
- singleton talents for if we do anything wacky later
- specializations which let you adapt to skills *faster* mid round

skills are going to be tied on the /datum/mind level.

see [skills on our hackmd](https://hackmd.io/EQXpS4TBTDOGtdp03E6CpQ?view) for more information

# This entire PR is config-disabled and is just going to add backend, character setup, etc. You are not affected yet.

:cl:
add: skills customization readded; make sure to set them up.
/:cl:

### CODE REVIEW

**Comment by silicons** [comment]
Oops, all featurecreep!

File: code/modules/preferences/preference_setup/skills/skills.dm [hunk 1]
```diff
@@ -1,8 +1,39 @@
-// This file has been UNTICKED on purpose, as it is not in use.
+// TODO: this file is still commented out, because we will not even *SHOW* it until skills implementation is near critical mass.
```
Comment (TheLordME) [inline code/modules/preferences/preference_setup/skills/skills.dm:1]: needs to be done before merge or for follow up PRs?

File: code/modules/overmap/disperser/disperser_console.dm [hunk 2]
```diff
@@ -150,7 +150,7 @@
 		data["range"] = range
 		data["next_shot"] = round(get_next_shot_seconds())
 		data["nopower"] = !data["faillink"] && (!front.powered() || !middle.powered() || !back.powered())
-		data["skill"] = user.get_skill_value(core_skill) > skill_offset
+		data["skill"] = TRUE // todo: skills
```
Comment (TheLordME) [inline code/modules/overmap/disperser/disperser_console.dm:153]: todo (:

**Review by TheLordME — APPROVED** [review]

File: code/modules/ghostroles/instantiator.dm [hunk 3]
```diff
@@ -159,9 +159,11 @@
 /datum/ghostrole_instantiator/human/player_static/Create(client/C, atom/location, list/params)
 	var/mob/living/carbon/human/H = ..()
 	var/list/errors = list()
+	// todo: respect warnings; we ignore them right now so we don't block joins.
```
[the remark this answers is not in the document]
Comment (silicons) [reply code/modules/ghostroles/instantiator.dm:162]: todo later

File: code/modules/overmap/disperser/disperser_console.dm (hunk 2 above)
Comment (silicons) [reply code/modules/overmap/disperser/disperser_console.dm:153]: yeah this kind of just tells the interface that the user is skilled enough & disperser is now hardcoded to work (as opposed to the old thing which hardcoded the skills to always be at max, which works too but was kind of just as silly)

File: code/modules/preferences/preference_setup/skills/skills.dm (hunk 1 above)
Comment (silicons) [reply code/modules/preferences/preference_setup/skills/skills.dm:1]: todo for followup prs, i'll need to end up making an admin interface for this too

**Review by TheLordME — APPROVED** [review]

**Pull request merged**